bff-nav_ansatt/src/test/kotlin/no/nav/amt/tiltak/bff/nav_ansatt/EndringsmeldingControllerTest.kt

```diff
@@ -0,0 +1,149 @@
+package no.nav.amt.tiltak.bff.nav_ansatt
+
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import no.nav.amt.tiltak.bff.nav_ansatt.dto.DeltakerDto
+import no.nav.amt.tiltak.common.auth.AuthService
+import no.nav.amt.tiltak.core.domain.tiltak.Endringsmelding
+import no.nav.amt.tiltak.core.port.DeltakerService
+import no.nav.amt.tiltak.core.port.EndringsmeldingService
+import no.nav.amt.tiltak.core.port.NavAnsattService
+import no.nav.amt.tiltak.core.port.TiltaksansvarligAutoriseringService
+import no.nav.amt.tiltak.test.database.data.TestData.BRUKER_1
+import no.nav.amt.tiltak.test.database.data.TestData.BRUKER_SKJERMET
+import no.nav.amt.tiltak.test.database.data.TestData.DELTAKER_1
+import no.nav.amt.tiltak.test.database.data.TestData.DELTAKER_1_STATUS_1
+import no.nav.amt.tiltak.test.database.data.TestData.GJENNOMFORING_1
+import no.nav.amt.tiltak.test.database.data.TestData.createDeltakerInput
+import no.nav.amt.tiltak.test.database.data.TestData.createStatusInput
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import java.time.LocalDate
+import java.time.ZonedDateTime
+import java.util.UUID
+
+class EndringsmeldingControllerTest {
```

Comment:
Hva er fordelen med å mocke ut denne isteden for å bruke integrasjonstestene?

Reply:
Jeg har lagt inn en integrasjonstest nå. Det ble litt tungvint iom at man ikke får deserialisert innhold, så har bare oppdatert expected json data også bør man se på oppsettet med å asserte på jsonblob senere. Har lagt inn enhetstester på de forskjellige scenarioene iom at det går raskere å kjøre enn å spinne opp hele applikasjonen for å teste ting som kan testes isolert.